source/Irrlicht/COGLES2Driver.cpp

```diff
@@ -2099,6 +2099,15 @@ COGLES2Driver::~COGLES2Driver()
 	}
 
 
+	void COGLES2Driver::setViewPortRaw(const core::rect<s32>& vp)

```

Comment:
This should be a `core::dimension2d<u32>`, I don't see this being called with negative values anywhere

Reply:
(For that matter, `setViewPort` should be a core::rect<u32>, but who said Irrlicht made sense?)

Fixed.